src/screens/ContactScreen/index.js

```diff
@@ -129,14 +138,15 @@ const ContactScreen = ({navigation}) => {
       username={item.username}
       followed={extendedState.followed}
       userid={item.user_id}
-      onPress={() => handleSelected(item)}
+      isAnon={sourceScreen === ANONYMOUS}
+      onPress={() => handleSelected(item, sourceScreen)}
     />
   );
   // }
   const handleSelected = (value) => {
-    const copyFollowed = [...followed];
-    const copyUsername = [...usernames];
-    const copyUsers = [...selectedUsers];
+    const copyFollowed = sourceScreen === ANONYMOUS ? [] : [...followed];
+    const copyUsername = sourceScreen === ANONYMOUS ? [] : [...usernames];
+    const copyUsers = sourceScreen === ANONYMOUS ? [] : [...selectedUsers];
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here seem to be handling the case where the source screen is `ANONYMOUS` differently than other cases. This looks fine, but make sure that the behavior in the `ANONYMOUS` case is as expected and doesn't introduce any unexpected side effects.



<!-- This is an auto-generated comment by OSS CodeRabbit -->